### ISSUE TITLE
How to place a new order using questrade api? 

### ISSUE THREAD

**Issue by marvelmaester** [tgardiner/questrade_api#13]
Hi Tom, Is it possible to place a new order using code? I only see a provision to retrieve existing orders using the questrade api, it is mentioned in the following Questrade documentation - https://www.questrade.com/api/documentation/rest-operations/account-calls/accounts-id-orders.

But is it also possible to create a new order using questrade api? Looking forward to your response. Thank you.

**Comment by tgardiner** [comment]
Please see #10

**Comment by marvelmaester** [comment]
Thank you Tom. Since if we are not able to place orders using the questrade api, what else can we use it for. For instance, will you be bale to tell us in two to three sentences what kind of purpose your code in this repository helps with? I am trying to understand the applications of questrad api and for what basic functions I can use it to automate. Thank you.

**Comment by saskatchewancatch** [comment]
I explored this question before.

The Questrade REST API is not well-maintained.  Questrade doesn’t even
answer professional emails addressed to their API support department. It is
strictly a REST API for read operations.

Questrade does have a C++ SDK on its website but that allows orders to be
placed and other execute operations. However, the last I tried it, even
their supplied example program wouldn’t run.

This particular Github project is basically a Python convenience module for
their REST API. It works well and helped me implement a script that scrapes
prices which I used to manually execute arbitrage tricks on the US/TSX
exchanges.

I highly discourage using the Questrade API for anything besides simple
read ops on your account and market quotes.
